docs/src/pages/guides/composition/ListRouter.js

```diff
@@ -81,17 +59,17 @@ export default function ComponentProperty() {
             <Typography gutterBottom>Current route: {location.pathname}</Typography>
           )}
         </Route>
-        <div className={classes.lists}>
+        <Paper elevation={0}>
           <List component="nav" aria-label="main mailbox folders">
```

Comment:
Just seeing this. Shouldn't you use `<nav><ul /></nav>`? This currently results in `<nav><li /></nav>` which does not have list semantics as far as I know.

Reply:
👍 Matt has done the same feedback, I have only fixed the markdown, didn't realize we had something in the source too.